vars/daosLatestVersion.groovy

```diff
@@ -58,26 +84,10 @@ String call(String next_version='1000', String distro=null) {
         }
     }
 
-    String v = null
-    try {
-        v = sh(label: 'Get RPM packages version',
-               script: '$(command -v dnf) --refresh repoquery --repofrompath=daos,' + env.ARTIFACTORY_URL +
-                       '/artifactory/daos-stack-daos-' + distro2repo(_distro) + '-x86_64-stable-local/' +
-                     ''' --repoid daos --qf %{version}-%{release} --whatprovides 'daos < ''' +
-                                  _next_version + '''' |
-                              rpmdev-sort | tail -1''',
-               returnStdout: true).trim()
-    /* groovylint-disable-next-line CatchException */
-    } catch (Exception e) {
-        sh(label: 'Get debug info',
-           script: 'hostname; pwd; df -h /var/cache; cat /etc/os-release')
-        println('Error getting latest daos version.')
-        throw e
-    }
+    String v = getLatestVersion()
 
-    if (!v) {
-        return ''
+    if (v) {
+        return v
     }
-
-    return v.replace(rpmDistValue(_distro), '')
+    return getLatestVersion('archive')
```

Comment:
I think this can be condensed down into (GitHub may mangle this suggestion as it doesn't do very well with suggestions on unchanged lines -- hopefully it's understandable):

```suggestion
    return getLatestVersion() || getLatestVersion('archive')
```

Reply:
Done

Reply:
Let's make it in more straitforward way.